### PR TITLE
Add organisation related use cases

### LINKED ISSUE #24: Add organisation related use cases to the Developer Guide

### PR DESCRIPTION
**Pull request by joshtyf**
Added organisation related use cases to the DG

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -471,7 +467,98 @@ MSS:
...
+**Actor: User**
+
+MSS:
+
+1. User creates a flashcard.
+2. QuickCache shows a list of existing tags.
+3. QuickCache asks the user to type in the tags one by one.
+4. User types in the tags one by one.
+5. User signals that he is finished.
+5. QuickCache adds the tags to the flashcard.
```
Comment (josiahkhoo) [inline docs/DeveloperGuide.md]: I think this should be number 6

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -471,7 +467,98 @@ MSS:
...
+**Extensions:**
+
+* 3a. User wishes to remove a tag that he entered.
+	* 3a1. User inputs the remove command to remove the tag.
+
+Use case resumes at step 3.
+
+* 5a. Some of the tags are new tags
+	* QuickCache stores and remembers the tags.
+	*
```
Comment (josiahkhoo) [inline docs/DeveloperGuide.md]: Empty point

**Review by josiahkhoo — CHANGES_REQUESTED** [review]
Looks good, just a few typos to update

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (joshtyf) [reply docs/DeveloperGuide.md]: fixed in the latest commit!

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (joshtyf) [reply docs/DeveloperGuide.md]: fixed in the latest commit!

**Review by josiahkhoo — APPROVED** [review]
Looks good, thanks Josh!

**Pull request merged**